pages/api/tool/[id]/module/post.ts

```diff
@@ -0,0 +1,18 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { Tool } from "../../../models/tool";
+import post from "../../../module/post";
+
+const toolPost = async (
+    req: NextApiRequest,
+    res: NextApiResponse,
+): Promise<void> => {
+    const { id } = req.query;
+    const tool = await Tool.findById(id).exec();
+    if (!tool)
+        return res
+            .status(404)
+            .send({ error: "The module with the given ID was not found." });
+    post(req, res);
```

Comment:
you dont use the tool at all here? Should you put the tool id somewhere in the request body before passing it off to the module post function?